docs/plugins/inputs/kafka.asciidoc

```diff
@@ -424,12 +424,12 @@ The amount of time to wait before attempting to retry a failed fetch request
 to a given topic partition. This avoids repeated fetching-and-failing in a tight loop.
 
 [id="plugins-{type}s-{plugin}-sasl_jaas_config"]
-===== `sasl_jaas_config`
+===== `sasl_jaas_config` 
 
```

Comment:
Only changes appear to be white space at the end of lines.  We don't need this, but I can live with it if keeping this file as part of the PR is the best approach.